src/server/frontend_wayland/wayland_connector.cpp

```diff
@@ -597,6 +597,8 @@ mf::WaylandConnector::WaylandConnector(
         };
 
         this->wayland_display = wayland_display;
+    } else {
```

Comment:
We use more whitespace around here:
```suggestion
    }
    else
    {
```